pattern-library/html/boilerplate-blank-RTL.html

```diff
@@ -0,0 +1,54 @@
+<!--
+edX Pattern Library: Boilerplate HTML Template: Header + Footer (RTL layout direction!)
+
+NOTES:
+
+* base template with details on how to handle primary/secondary assets, metadata, and basic view-level UI (on canvas vs. conditional and off-canvas elements)
+* right-to-left (RTL) layout configuration have been added
+* see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-HTML#right-to-left-rtl-support for more information
+-->
+
+<!DOCTYPE html>
+<html class="no-js" lang="en" dir="rtl">
+    <head dir="rtl">
+        <meta charset="utf-8">
+        <!-- always force latest IE rendering engine or request chrome rrame -->
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
```

Comment:
If you're aiming for strict validation here, it's best to define these in the headers for the document, not in the page source itself.  IE=edge,chrome=1 is not a valid value for the content attribute here.